code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm

```diff
@@ -137,6 +137,12 @@
 		if(A.buffed || (A.type in banned_mobs) || A.stat)
 			to_chat(user, "<span class='warning'>Something's interfering with [src]'s effects. It's no use.</span>")
 			return
+		if(A.mob_biotypes & MOB_ROBOTIC)
+			to_chat(user, "<span class='warning'>[src] doesn't work on robotic lifeforms! That makes no sense!</span>")
+			return
+		if(A.mob_biotypes & MOB_SPIRIT)
+			to_chat(user, "<span class='warning'>You can't apply [src] to [A], they're a spirit!</span>")
+			return
```

Comment:
these aren't really very good criteria as to whether or not fugu glands would be busted if used on something. oh no, a shade can deal slightly more damage, the horror.

you could maybe replace them with a godmode check and/or a megafauna check or something, but tbh, it's probably better to account for this on a case-by-case basis, such as by actually putting mobs in that banned_mobs list (or by making your new simplemobs account for the idea of someone giving them the ability to attack somehow, including via spells or the like).

Reply:
basically, the problem here is floyd adding an easily-accessible nigh-invincible mob to the station and expecting players to NOT immediately abuse it, not the fugu glands or the mind transference potions.

you didn't even account for sentience potions, smh my head my head.

Reply:
> these aren't really very good criteria as to whether or not fugu glands would be busted if used on something. oh no, a shade can deal slightly more damage, the horror.
> 
> you could maybe replace them with a godmode check and/or a megafauna check or something, but tbh, it's probably better to account for this on a case-by-case basis, such as by actually putting mobs in that banned_mobs list (or by making your new simplemobs account for the idea of someone giving them the ability to attack somehow, including via spells or the like, _floyd_).

who CARES IT MAKES NO SENSE HOW CAN YOU APPLY A GLAND TO A SPIRIT

Reply:
you can punch shades, I don't see why you shouldn't be allowed to smear glands on them